.travis.yml

```diff
@@ -14,18 +14,18 @@ branches:
     - stable
     - /^feature-.*$/
 
+xcode_workspace: ./catalog/MDCCatalog.xcworkspace
+xcode_scheme: MDCCatalog
+
 env:
   global:
   - LC_CTYPE=en_US.UTF-8
   - LANG=en_US.UTF-8
   matrix:
-    - DESTINATION="platform=iOS Simulator,name=iPhone 7,OS=10.3.1" SDK="iphonesimulator10.3.1"
-    # The iPhone 5 was chosen because it is a 32-bit device.
-    # 8.1 was chosen because it is the earliest iOS offered here.
-    - DESTINATION="platform=iOS Simulator,name=iPhone 5,OS=8.1" SDK="iphonesimulator8.1"
```

Comment:
If we have 32-bit specific code, that's probably more important than the percentage.